### ISSUE TITLE
Gradle namespace

### ISSUE THREAD

**Issue by TheCarpetMerchant** [kineapps/flutter_archive#73]
Please provide the needed adjustments for the use of Gradle 8.

**Comment by paulocoutinhox** [comment]
Please add the things for Gradle 8.

**Comment by paulocoutinhox** [comment]
Also, im getting:

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':flutter_archive:compileDebugKotlin'.
> 'compileDebugJavaWithJavac' task (current target is 1.8) and 'compileDebugKotlin' task (current target is 17) jvm target compatibility should be set to the same Java version.
  Consider using JVM toolchain: https://kotl.in/gradle/jvm/toolchain
```

**Comment by r280822a** [comment]
Try
```
flutter_archive:
  git:
    url: https://github.com/kineapps/flutter_archive.git
```

In pubspec.yaml

**Comment by kinex** [comment]
Fixed in 6.0.0
